Makefile

```diff
@@ -2,7 +2,19 @@
 # Makefile by Roberto M.F. (Roboe)
 # https://github.com/WeAreFairphone/flashabe-zip_emojione
 
-SHELL     := /bin/bash
+SHELL       := /bin/bash
+
+# Dependencies
+CURL        := $(shell command -v curl 2>&1)
+ZIP         := $(shell command -v zip 2>&1)
+UNZIP       := $(shell command -v unzip 2>&1)
+ifeq ($(uname -s),"Darwin")
```

Comment:
This comparison doesn't seem to work at all as on macOS the `else` path is taken.

Reply:
$ if [ $(uname -s) = 'Darwin' ] ; then echo true; else echo false ; fi
true
$ if [ $(uname -s) = 'Linux' ] ; then echo true; else echo false ; fi
false